### DISCUSSION TITLE
Metropolis

### DISCUSSION THREAD

**Discussion by lambda-11235** [redeclipse/base#1177] [Maps and Content Showcase]
A ctf map. One of my main concerns is performance, so if someone with a potato for a computer could comment that would be great.

![20191107233442](https://user-images.githubusercontent.com/16354815/68458472-c12fce00-01b7-11ea-9c70-4bf8734cc3f9.png)
![20191107233456](https://user-images.githubusercontent.com/16354815/68458477-c3922800-01b7-11ea-8921-247a9206c44e.png)
![20191107233514](https://user-images.githubusercontent.com/16354815/68458482-c68d1880-01b7-11ea-85f1-aa65b6cad29f.png)

r103:
[metro.zip](https://github.com/red-eclipse/world/files/3459495/metro.zip)

r145:
[metro.zip](https://github.com/red-eclipse/world/files/3463048/metro.zip)

r171:
[metro.zip](https://github.com/red-eclipse/world/files/3479814/metro.zip)

r203:
[metro.zip](https://github.com/red-eclipse/world/files/3504164/metro.zip)

r220:
[metro.zip](https://github.com/redeclipse/world/files/3640391/metro.zip)

r276:
[metro.zip](https://github.com/redeclipse/world/files/3668367/metro.zip)
~[lambda-11235.zip](https://github.com/redeclipse/world/files/3668375/lambda-11235.zip)~
[lambda-11235.zip](https://github.com/redeclipse/world/files/3668520/lambda-11235.zip)

r278:
[metro.zip](https://github.com/redeclipse/world/files/3668528/metro.zip)
[lambda-11235.zip](https://github.com/redeclipse/world/files/3668527/lambda-11235.zip)

r344:
[metro.zip](https://github.com/redeclipse/world/files/3823476/metro.zip)

**Comment by lambda-11235** [comment]
Windows may need to be redone. RE can't handle that much geometry. It slows map loading down to a crawl (>15s).

**Comment by no-lex** [comment]
Few things:
 - Zip files should be completely flat so the game can load them (this means no folders at all)
 - The reason the map takes so long to load and remip is the `ond` count (octree nodes): this map has 6,489,776 (compared to *error* with 152,472 or *auster* with 170,992)
 - The occlusion on this map is quite terrible, with the windows allowing clear visibility between bases. You will need to reduce the occlusion issues (perhaps by blinding some/most windows and adding blocking geometry) in order to improve performance, as this map is already at the low end of what I'd consider acceptable.

**Comment by lambda-11235** [comment]
Current TODO list

- [ ] Fix problems with envmaps.
- [ ] Fix texturing on stairs in each teams building. Right now the textures are rather harsh against the concrete.
- [x] Add more room features to the four buildings surrounding the central one.

**Comment by lambda-11235** [comment]
The new update requires a custom mapmodel. Put lambda-11235.zip in `~/.redeclipse`. I'm not sure if unzipping is necessary (I don't think so).
